build.bat

```diff
@@ -11,6 +11,12 @@ if exist 8060d430.bin del 8060d430.bin > nul
 if exist 8070_430.bin del 8070_430.bin > nul
 if exist 8070d430.bin del 8070d430.bin > nul
 
+REM Claus  jetzt vor übersetzten löschen, da unten nicht mehr gelöscht

```

Comment:
Can you translate it ?

Reply:
New trial:
Claus will be deleted here, since no longer deleted below.
But the plan was not to change the batch file at all for common use, was done for my local build, only.
Can you reject it or what do I have to do?